imports/app/ImageSettings.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import Paper from 'material-ui/Paper';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+import ArrowDropRight from 'material-ui/svg-icons/navigation-arrow-drop-right';
+import GridControl from '../gridControl/GridControl.js';
+
+export default class ImageSettings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      subOption: '',
```

Comment:
@julieyao05 ,  may change to mongo style?

Reply:
Also, it is better to add the default value of "option" in this.state in this constructor.